packages/uikit/src/widgets/RoiCalculator/RoiCalculator.tsx

```diff
@@ -317,6 +328,25 @@ export function RoiCalculator({
     </Message>
   );
 
+  const tokenA = (currencyA ?? undefined)?.wrapped;
+  const tokenB = (currencyB ?? undefined)?.wrapped;
+  const isSorted = tokenA && tokenB && tokenA.sortsBefore(tokenB);
+  const leftPrice = Number(formatPrice(isSorted ? priceRange?.priceLower : priceRange?.priceUpper?.invert()));
+  const rightPrice = Number(formatPrice(isSorted ? priceRange?.priceUpper : priceRange?.priceLower?.invert()));
```

Comment:
the formatPrice can be '-' or '<0.0001'

Reply:
It should be feasible to use the formatPrice function with formatFractions without returning '-' or '<0.0001'.